app/src/app/components/Map.tsx

```diff
@@ -32,14 +32,19 @@ export const MapComponent: React.FC = () => {
   const hoverFeatureIds = useHoverFeatureIds();
   const document = useCreateMapDocument();
```

Comment:
fyi this mutation object being conflated with the store object seems to be a big source of confusion later on. I renamed the references to the store entry as `storeDocument` but we may instead want to rename this `mutateDocument` or something